db/seeds.rb

```diff
@@ -33,9 +33,9 @@
 		phone: "206-723-1076",
 		website: "www.ugm.org"
 	})
-SeattleUnionGospelMission.institution_detail = InstitutionDetail.new({
-		hours: "No Hours Set"
-	})
+# SeattleUnionGospelMission.institution_detail = InstitutionDetail.new({
+# 		hours: "No Hours Set"
+# 	})
```

Comment:
We should probably just delete this entire file. There's not really a point in having it in here.

The amenities seed doesn't work very well unless you drop your database every time, which isn't something we can do on production. I wrote a little script I've been using here.
https://github.com/collaborations/capstone/blob/master/lib/tasks/seed_amenities.rake